src/SFML/System/Android/SuspendAwareClock.cpp

```diff
@@ -22,31 +22,24 @@
 //
 ////////////////////////////////////////////////////////////
 
-#ifndef SFML_SLEEPIMPLUNIX_HPP
-#define SFML_SLEEPIMPLUNIX_HPP
-
 ////////////////////////////////////////////////////////////
 // Headers
 ////////////////////////////////////////////////////////////
-#include <SFML/Config.hpp>
-#include <SFML/System/Time.hpp>
+#include <SFML/System/Android/SuspendAwareClock.hpp>
+#include <ctime>
 
+#ifdef CLOCK_BOOTTIME
```

Comment:
I stumbled upon [this mailing list entry](https://lwn.net/Articles/428176/), did you find proper documentation about it?

Also, isn't it an error if `CLOCK_BOOTTIME` is not defined? Maybe we should directly use `#error`, because now the user would get some compile errors about missing symbols instead (unless I'm missing something).

Reply:
Linux man pages describe this clock well. https://linux.die.net/man/2/clock_gettime

I have seen mailing lists about trying to merge `CLOCK_MONOTONIC` and `CLOCK_BOOTTIME` (which was accepted even by Linus Torvalds) but later on, it proved to be a mistake as various components had already been unprepared for suspend-awareness which caused bugs (e.g. `systemd`) and so [they reverted it](https://www.spinics.net/lists/linux-tip-commits/msg43709.html).

> Also, isn't it an error if CLOCK_BOOTTIME is not defined? 

You're probably right. I will update.